optaplanner-core/src/test/java/org/optaplanner/core/impl/score/director/DummyEasyScoreCalculator.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.core.impl.score.director;
+
+import org.optaplanner.core.api.score.buildin.simple.SimpleScore;
+import org.optaplanner.core.api.score.calculator.EasyScoreCalculator;
+import org.optaplanner.core.impl.testdata.domain.constraintconfiguration.TestdataConstraintConfigurationSolution;
+
+public final class DummyEasyScoreCalculator
+        implements EasyScoreCalculator<TestdataConstraintConfigurationSolution, SimpleScore> {
+
+    @Override
+    public SimpleScore calculateScore(TestdataConstraintConfigurationSolution solution) {
+        SimpleScore constraintWeight = solution.getConstraintConfiguration().getFirstWeight();
+        return constraintWeight.multiply(solution.getEntityList().size());
```

Comment:
Same story here. SolverconfigTest already has a dummy which really does nothing. I'd call it TestdataSolutionConstraintProvider.